docs/src/Popover.doc.js

```diff
@@ -10,8 +10,8 @@ const card = (c) => cards.push(c);
 
 card(
   <PageHeader
-    name="Flyout"
-    description="Flyouts are similar to modals. They’re an alternative when there is less content to display,
+    name="Popover"
+    description="Popovers are similar to modals. They’re an alternative when there is less content to display,
```

Comment:
We should probably add a temporary note here that the component was formerly known as Flyout, something like "Popover (formerly known as Flyouts) are similar to modals."

Reply:
Got it.

Reply:
DONE